release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -96,15 +106,12 @@ def parse_args():
     if args.data_root is None:
         # use default datasets if data root is not provided
         if args.file_type == "image":
-            # 1GB ragged dataset
-            args.data_root = "s3://imagenetmini1000/1gb/train/"
-
-            # Alternative larger dataset
-            # args.data_root = "s3://air-example-data-2/10G-image-data-synthetic-raw"  # noqa: E501
-
+            args.data_root = get_prop_raw_image_paths(
+                num_workers=args.num_workers, target_worker_gb=args.target_worker_gb
+            )
         elif args.file_type == "parquet":
-            args.data_root = (
-                "s3://air-example-data-2/20G-image-data-synthetic-raw-parquet"
+            args.data_root = get_prop_parquet_paths(
+                num_workers=args.num_workers, target_worker_gb=args.target_worker_gb
```

Comment:
also allow reading the whole dataset? maybe `--target-worker-gb=full`

Reply:
if you specify an explicit `--data-root <path>`, it will skip proportional allotment of paths. is that good enough of a workaround?